src/Makefile

```diff
@@ -65,20 +65,22 @@ ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
 ifeq ($(USE_POLLY_ACC),1)
-LLVMLINK += -lPollyPPCG
+LLVMLINK += -lPollyPPCG -lGPURuntime
 FLAGS += -DUSE_POLLY_ACC
-ifeq ($(CUDALIB_INCLUDE_DIR),)
-$(error CUDALIB_INCLUDE_DIR not set: If compiling with USE_POLLY_ACC, user must provide the path to CUDA header files)
-endif
-FLAGS += -I$(CUDALIB_INCLUDE_DIR)
-FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools
+FLAGS += -I$(shell $(LLVM_CONFIG_HOST) --src-root)/tools/polly/tools # Required to find GPURuntime/GPUJIT.h
 endif
 endif
 else
 SRCS += anticodegen
 LLVM_LIBS := support
 endif
 
+$(build_shlibdir)/libGPURuntime.$(SHLIB_EXT):
+ifeq ($(USE_POLLY_ACC), 1)
+# Place libGPURuntime such that it's accessible under libjulia's RPATH
+	@cp $(shell $(LLVM_CONFIG_HOST) --libdir)/libGPURuntime.$(SHLIB_EXT) $(build_shlibdir)
+endif
```

Comment:
Not just that, It would be beneficial while developing Julia and LLVM (and Polly ;-)). This is what I have,
```
$ ls
julia_src  llvm_build  llvm_src  polly_src
```
Some of the benefits I've felt,
1. Helps me to be mindful of each step of the build. e.g I know that issuing a `make` in julia_src/ will not build llvm.
2. If the LLVM and Polly repos are going to used to submit patches, having them outside the julia source-tree would be better.